queries/inko/highlights.scm

```diff
@@ -0,0 +1,168 @@
+; Brackets and operators
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+[
+  "+"
+  "-"
+  "/"
+  "*"
+  "**"
+  "%"
+  "<"
+  ">"
+  "<="
+  ">="
+  "<<"
+  ">>"
+  ">>>"
+  "&"
+  "|"
+  "^"
+  "=="
+  "!="
+] @operator
```

Comment:
Can we get `"->"` here too? Also `","`, `"."`, and `":"` as `@punctuation.delimiter`

Reply:
Also looks like assignment operators like `"+="`, `"-="`, etc. are missing

Reply:
@ribru17 `->` isn't an actual operator though, it's used to signal the return type of a function, i.e. `fn foo -> Int { ... }`, or does that also count as an operator?

Reply:
Ah, I see sorry. Maybe it would be better as `@punctuation.delimiter`